docs/sql/project.mdx

```diff
@@ -3,303 +3,61 @@ title: MindsDB Projects
+Projects store [all MindsDB schema objects](/sql/table-structure#the-information-schema-database) except for handlers, connected data sources, and configured AI/ML engines. That is, projects can store models, views, jobs, triggers, agents, skills, knowledge bases, and chatbots.
 
-### Creating `PROJECTS`
+MindsDB provides the default `mindsdb` project where all objects created without defining a project are stored.
 
-You can create projects to store your models, making the structure like this:
+## Working with MindsDB Projects
 
-```
-project_alpha
-├─ models
-├─ model_a
-├─ model_b
-project_beta
-├─ models
-├─ model_c
-```
+### Create a Project
 
-Here is how you create a project:
+Use the below command to create a project.
 
 ```sql
-CREATE PROJECT project_alpha;
+CREATE PROJECT project_name;
 ```
 
 <Tip>
 Use lower-case letters for a project name.
 </Tip>
 
-### Viewing `PROJECTS`
-
-There are two ways you can list all your projects:
-
-1. Use the `SHOW DATABASES` command:
-
-    ```sql
-    SHOW DATABASES WHERE type = 'project';
-    ```
-
-    On execution, we get:
-
-    ```sql
-    +----------------------+
-    | Database             |
-    +----------------------+
-    | mindsdb              |
-    | project_alpha        |
-    | project_beta         |
-    +----------------------+
-    ```
-
-    Please note that `mindsdb` is the default project.
-
-2. Use the `SHOW FULL DATABASES` command to get more details:
-
-    ```sql
-    SHOW FULL DATABASES WHERE type = 'project';
-    ```
+### List All Projects
 
-    On execution, we get:
-
-    ```sql
-    +----------------------+----------+-----------+
-    | Database             | TYPE     | ENGINE    |
-    +----------------------+----------+-----------+
-    | mindsdb              | project  | [NULL]    |
-    | project_alpha        | project  | [NULL]    |
-    | project_beta         | project  | [NULL]    |
-    +----------------------+----------+-----------+
-    ```
-
-    <Tip>
-    To view all connected data sources, specify `type = 'data'` in the `WHERE` clause.
-    </Tip>
-
-### Dropping `PROJECTS`
-
-Here is how you can remove a project:
+Use the below command to list all projects.
 
 ```sql
-DROP PROJECT project_alpha;
+SHOW [FULL] DATABASES
+WHERE type = 'project';
 ```
 
-On execution, we get:
+### Create an Object within a Project
+
+Use the below command template to create an object within a project.
 
 ```sql
-Query OK, 0 rows affected (x.xxx sec)
+CREATE <OBJECT> project_name.object_name
+...;
 ```
 
 <Note>
-   **Cannot Drop a Project**
-
-   Please note that if your project stores at least one object, it cannot be removed. In this case, you should first drop all the jobs and tables, such as models and views, belonging to this project, and then, you can remove the project.
-   Please see the [Example](#example) section for details.
+Please note that if you use the same name for a model and a view stored in one project, then MindsDB adds `_view` to the view name.
```

Comment:
do we have a such feature (adding _view to name) ? I don't know about it

Reply:
I was informed about it a while back, maybe it's obsolete now.
Let me remove it.